source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -308,7 +308,7 @@ - (void)processTextConcurrently:(std::shared_ptr<BaseCardElement> const &)textEl
             NSDictionary *data = nil;
 
             // use Apple's html rendering only if the string has markdowns
-            if(markDownParser->HasHtmlTags()) {
+            if(markDownParser->HasHtmlTags() || markDownParser->IsEscaped()) {
```

Comment:
do we know if this behavior shows in other platforms?

Reply:
the issuer mentioned that he or she tested it also on Desktop and Android, and they don't reproduce, but i'm not certain which version of Destop sample app they are referring to.

---
In reply to: [215794246](https://github.com/Microsoft/AdaptiveCards/pull/1865#discussion_r215794246) [](ancestors = 215794246)